nlmod/mdims/resample.py

```diff
@@ -10,6 +10,9 @@
 from scipy.interpolate import griddata
 import rasterio
 from scipy.spatial import cKDTree
+from shapely.geometry import Polygon
+from shapely.affinity import affine_transform
+from affine import Affine
```

Comment:
Extra dependency, add to setup.py

Reply:
Will add it. It is a probably a required dependency of rioxarray and ratserio as well.